src/main/java/duke/bin/common/Time.java

```diff
@@ -0,0 +1,50 @@
+package duke.bin.common;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Time class to hold value of time with correct display.
+ */
+public class Time {
```

Comment:
I like this class. It's pretty useful for Duke to understand time :)

Reply:
Thanks!